atomic_reactor/build.py

```diff
@@ -187,7 +187,7 @@ def set_base_image(self, base_image, parents_pulled=True, insecure=False):
         logger.info("setting base image to '%s'", base_image)
         self.base_image = ImageName.parse(base_image)
         self.original_base_image = self.original_base_image or self.base_image
-        self.parent_images[str(self.original_base_image)] = base_image
+        self.parent_images[str(self.original_base_image)] = self.base_image
```

Comment:
confused. shouldn't this be without the `str()` ?

There seem to be some more bits in this file that need updates like
https://github.com/projectatomic/atomic-reactor/blob/6ae4ec329f7b81eb3ffcc62734d392c2165cc259/atomic_reactor/build.py#L183-L184
... will initialize it with strings won't it?

Reply:
Fixed.